ci/prow-entrypoint.sh

```diff
@@ -124,9 +124,9 @@ kola_test_qemu() {
     # Skip Secure Boot tests on SCOS for now
     # See: https://github.com/openshift/os/issues/1237
     if rpm-ostree compose tree --print-only "${manifest}" | jq -r '.packages[]' | grep -q "centos-release"; then
-        cosa kola --basic-qemu-scenarios --output-dir ${ARTIFACT_DIR:-/tmp}/kola-basic
+        cosa kola --output-dir ${ARTIFACT_DIR:-/tmp}/kola-basic
     else
-        cosa kola --basic-qemu-scenarios --skip-secure-boot --output-dir ${ARTIFACT_DIR:-/tmp}/kola-basic
+        cosa kola --skip-secure-boot --output-dir ${ARTIFACT_DIR:-/tmp}/kola-basic
     fi
```

Comment:
This would still do a separate kola run. We need to instead translate it to just changing the arguments that would be passed to `cosa kola run` below (line 131) for whether or not to denylist the Secure Boot tests.

Reply:
Sorry I don't quite understand :/ Can we chat tmr ?

Reply:
So the current behaviour (without your patch) is to do two runs: first the basic scenarios, and then the full kola run. This is similar to the previous behaviour in coreos-ci-lib.

With the cosa change, we don't want to do that first run anymore since the basic tests are part of the full kola run. So that if-statement is just about whether or not we will need to denylist the Secure Boot tests in the full run.

But actually, for this PR to pass CI (and CI in https://github.com/coreos/coreos-assembler/pull/3652), we could do a similar thing to what we did in coreos-ci-lib, i.e. keep the current logic of two runs if old cosa, and otherwise a single run.

Reply:
Ah yes, that makes sense, for one reason or another the first comment was bouncing off my noggin.

Reply:
What do we do about the notes? *.md files that are edited, should I add a comment about it being 1 thing in one version and another in the new version?